tensorrt_llm/serve/postprocess_handlers.py

```
@@ -404,9 +405,43 @@ def from_request(cls, request: CompletionRequest):
 
+def create_completion_logprobs(token_ids: List[int],
+                               tokenizer: TransformersTokenizer,
+                               logprobs: List[float] | TokenLogprobs,
+                               initial_offset: int = 0) -> CompletionLogProbs:
+    assert len(token_ids) == len(logprobs), \
+            "token_ids and logprobs have different lengths"
+    text_offset = []
+    token_logprobs = []
+    top_logprobs_list = []
+    tokens = []
+    for token_id, logprob in zip(token_ids, logprobs):
```

Ruff (0.14.13)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)